File: Data Structures & Algorithms/sqrtx/submission-1.cpp
class Solution {
public:
    int mySqrt(int x) {
      long long i=0;
      while(1){
        if(i*i==x){
           return i;
        }else if(i*i>x){
            return i-1;
        }
        i++;
      } 
      return -1; 
    }
};

File: Data Structures & Algorithms/find-words-that-can-be-formed-by-characters/submission-1.py
class Solution:
    def countCharacters(self, words: List[str], chars: str) -> int:
        count=Counter(chars)
        sum=0
        for word in words:
            word_count=Counter(word)
            s=True
            for ch in word:
                if count[ch]<word_count[ch]:
                    s=False
                    break
            if s==True:
                sum+=len(word)
        return sum
        